src/qtism/runtime/expressions/operators/Utils.php

```diff
@@ -331,23 +331,24 @@ public static function lastPregErrorMessage()
      * Prepare an XSD Regular Expression pattern into a PCRE compliant one.
      *
      * @param string $pattern
+     * @param string $pcreFlags
      * @return string
      */
-     public static function prepareXsdPatternForPcre(string $pattern): string
-     {
-         // XML schema always implicitly anchors the entire regular expression
-         // Neither caret (^) nor dollar ($) sign have special meaning so they are
-         // considered as normal characters.
-         // see http://www.regular-expressions.info/xml.html
-         $pattern = self::withoutStringAnchors($pattern);
-         $pattern = self::escapeSymbols($pattern, ['$', '^']);
-         $pattern = self::pregAddDelimiter('^' . $pattern . '$');
-
-         // XSD regexp always case-sensitive (nothing to do), dot matches white-spaces (use PCRE_DOTALL).
-         $pattern .= 's';
-
-         return $pattern;
-     }
+    public static function prepareXsdPatternForPcre(string $pattern, string $pcreFlags = 'su'): string
+    {
+        // XML schema always implicitly anchors the entire regular expression
+        // Neither caret (^) nor dollar ($) sign have special meaning so they are
+        // considered as normal characters.
+        // see http://www.regular-expressions.info/xml.html
+        $pattern = self::withoutStringAnchors($pattern);
+        $pattern = self::escapeSymbols($pattern, ['$', '^']);
+        $pattern = self::pregAddDelimiter('^' . $pattern . '$');
+
+        // XSD regexp always case-sensitive (nothing to do), dot matches white-spaces (use PCRE_DOTALL).
+        $pattern .= $pcreFlags;
+
+        return $pattern;
+    }
```

Comment:
Hm, something went sideways with the indentation.
```suggestion
     public static function prepareXsdPatternForPcre(string $pattern, string $pcreFlags = 'su'): string
     {
         // XML schema always implicitly anchors the entire regular expression
         // Neither caret (^) nor dollar ($) sign have special meaning so they are
         // considered as normal characters.
         // see http://www.regular-expressions.info/xml.html
         $pattern = self::withoutStringAnchors($pattern);
         $pattern = self::escapeSymbols($pattern, ['$', '^']);
         $pattern = self::pregAddDelimiter('^' . $pattern . '$');

         // XSD regexp always case-sensitive (nothing to do), dot matches white-spaces (use PCRE_DOTALL).
         $pattern .= $pcreFlags;

         return $pattern;
     }
```

Reply:
Are you sure? When I first went in, this method had weird `4n+1` indentation, I only reformatted it to 4.

Reply:
Sorry for that, totally my bad here. All good with the PR!